frontend/workflows-lib/lib/utils/validationUtils.ts

```diff
@@ -0,0 +1,102 @@
+import { ScanRange } from "../types";
+
+export const isStrictPositiveInteger = (value: string): boolean =>
+  /^\d+$/.test(value) && Number(value) >= 0;
+
+export const parseExcludedScans = (input: string): number[] => {
+  const cleaned = input.replace(/[[\]]/g, "");
+
+  const parts = cleaned
+    .split(",")
+    .map((part) => part.trim())
+    .filter((part) => part !== "");
+
+  const resultSet = new Set<number>();
+
+  for (const part of parts) {
+    if (/^\d+$/.test(part)) {
+      resultSet.add(Number(part));
+    } else if (/^\d+-\d+$/.test(part)) {
+      const [startStr, endStr] = part.split("-");
+      if (
+        !isStrictPositiveInteger(startStr) ||
+        !isStrictPositiveInteger(endStr)
+      ) {
+        throw new Error("Invalid range");
+      }
+      const start = Number(startStr);
+      const end = Number(endStr);
+      if (start >= end) {
+        throw new Error("Invalid range");
+      }
+      for (let i = start; i <= end; i++) {
+        resultSet.add(i);
+      }
+    } else {
+      throw new Error("Values must be positive integers");
```

Comment:
To get to this point what the user could have entered anything and maybe they don't understand the syntax that they are supposed to use.

It may be better to explicitly specify the format that this input expects (currently a comma-separated list of positive integers).